subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync.c

```diff
@@ -277,10 +366,9 @@ static int prepare_cb(struct lll_prepare_param *p)
 
 	radio_tmr_aa_capture();
 
-	hcto = remainder_us +
-	       ((EVENT_JITTER_US + EVENT_TICKER_RES_MARGIN_US +
-		 lll->window_widening_event_us) << 1) +
-	       lll->window_size_event_us;
+	hcto = ((EVENT_JITTER_US + EVENT_TICKER_RES_MARGIN_US + lll->window_widening_event_us)
+		<< 1) +
+	       remainder_us + lll->window_size_event_us;
```

Comment:
Any reason to change the order?

Original order wass, the packet/event  timer takes `remainder_us` to apply the fine microsecond interval remainder, then window widening plus the jitters to be considered, finally the window size.

Reply:
Nope, just wanted to avoid constant updates of re-formatted code by clang-format. As it may be seen above, rearrangement didn't help either. 
Reverted to original form.